fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java

```diff
@@ -383,6 +384,7 @@ public BrokerFileSystem getS3AFileSystem(String path, Map<String, String> proper
                 conf.set(FS_S3A_ACCESS_KEY, accessKey);
                 conf.set(FS_S3A_SECRET_KEY, secretKey);
                 conf.set(FS_S3A_ENDPOINT, endpoint);
+                conf.set("fs.s3a.impl.disable.cache", "true");
```

Comment:
Please use constant value. Add add a comment to explain why need this config.

Reply:
If this property is not set to "true", FileSystem instance will be returned from cache which is not thread-safe and may cause 'Filesystem closed' exception when it is closed by other thread.

Reply:
@frwrdt 
You should add this comment in the source file to let others known not just the reviewer.

Reply:
ok